plugins/drude/tests/TestDrudeNoseHoover.h

```diff
@@ -131,7 +131,7 @@ void testWaterBox() {
     double frequencyDrude = 1000.0;
     int randomSeed = 100;
     DrudeNoseHooverIntegrator integ(temperature, frequency,
-                                    temperatureDrude, frequencyDrude, 0.0005,
+                                    temperatureDrude, frequencyDrude, 0.0004,
```

Comment:
Did this change accidentally creep in?

Reply:
That was intentional.  The changes caused this test to start failing, but on investigating I discovered it was just that the test was right on the edge of passing/failing, and any tiny perturbation to it could make it fail.  Reducing the time step slightly makes it much more robust.